manifests/todo-app.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: todo-app
+  name: todo-app
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: todo-app
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: todo-app
+    spec:
+      containers:
+      - image: eu.gcr.io/reload-oliver-onboarding/todo-app:${RELEASE_TAG}
+        name: todo-app
+        env:
+        - name: MYSQL_HOST
+          value: todo-db-svc
+        - name: MYSQL_USER
+          value: root
+        - name: MYSQL_PASSWORD
+          value: secret
+        - name: MYSQL_DB
+          value: todos
+        ports:
+        - containerPort: 3000
+        imagePullPolicy: IfNotPresent
```

Comment:
This can also be omitted if you want. Relevant: https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting